server/routes/api/team.js

```diff
@@ -0,0 +1,267 @@
+var router = require('express').Router(),
+    Team = require('../../db/model/Team.js'),
+    Application = require('../../db/model/Application.js'),
+    Responses = require('../../responses/api/team.js');
+
+// Handles get requests for /v1/teams
+router.get('/', function(req, res) {
+    Team.find({}, '-_id -__v')
+        .populate('members', 'full_name email avatar')
+        //.select('members.full_name')
+        .exec()
+        .then(teams => {
+            res.send({
+                status: true,
+                teams: teams
+            });
+        })
+        .catch(err => {
+            console.error(err);
+            res.status(500).send({
+                status: false,
+                message: Responses.UNKNOWN_ERROR
+            });
+        });
+});
+
+router.post('/', function(req, res) {
+    if (req.session.loggedIn) {
```

Comment:
This will only happen if the user is logged in. You probably don't need this check